packages/flutter_test/lib/src/_goldens_web.dart

```diff
@@ -60,17 +60,18 @@ class DefaultWebGoldenComparator extends WebGoldenComparator {
   Uri testUri;
 
   @override
-  Future<bool> compare(Element element, Size size, Uri golden) async {
+  Future<bool> compare(dynamic element, dynamic size, Uri golden) async {
```

Comment:
I just checked, the `element` being passed here is no longer being used, even internally. We could remove this parameter, and expand `size` to `width` and `height` to avoid having to use `dynamic`.

Reply:
even better!